File: backend-api/reservation-api/src/main/java/com/hcmute/reservation/model/dto/report/NoShowRateResponse.java
package com.hcmute.reservation.model.dto.report;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class NoShowRateResponse {
    private long totalOnline;
    private long totalWalkIn;
    private long totalAll;
    private long noShowCount;
    private double noShowRate;
}

File: desktop-staff/reservation-winforms/DTO/report/NoShowRateResponse.cs
using Newtonsoft.Json;

namespace reservation_winforms.DTO.report
{
    public class NoShowRateResponse
    {
        [JsonProperty("totalOnline")]
        public long TotalOnline { get; set; }

        [JsonProperty("totalWalkIn")]
        public long TotalWalkIn { get; set; }

        [JsonProperty("totalAll")]
        public long TotalAll { get; set; }

        [JsonProperty("noShowCount")]
        public long NoShowCount { get; set; }

        [JsonProperty("noShowRate")]
        public double NoShowRate { get; set; }
    }
}